app/common/update.py

```diff
@@ -334,9 +334,7 @@ def download_dat_files():
                         f.write(idx_request.content)
                     logger.success("Data files downloaded.")
                 else:
-                    logger.error(f"Failed to download data files.")
-                    input("Press ENTER to exit.")
-                    sys.exit()              
+                    logger.error(f"Failed to download data files. Clarity will continue without downloading.")    

```

Comment:
Don't need f-string formatting here since we aren't trying to access any variables in the string.